<?xml version="1.0" encoding="utf-8"?>
<launch>
  
  <node pkg="infinitam" type="infinitam" name="infinitam" output="screen">
    <!-- <param name="calibFile" value="/home/mgualti/mgualti/rosws/src/infinitam/calibration/structure.txt" /> -->
    <param name="calibFile" value="/home/baxter/rosws/src/infinitam/calibration/structure.txt" />
    <param name="mu" value="0.02" /> <!-- width of volume in voxels is mu / voxelSize -->
    <param name="maxW" value="100" />
    <param name="voxelSize" value="0.003" />
    <param name="viewFrustumMin" value="0.20" />
    <param name="viewFrustumMax" value="3.00" />
    <param name="noHierarchyLevels" value="5" />
    <param name="noRotationOnlyLevels" value="3" />
    <param name="depthTrackerICPThreshold" value="0.01" />
    <param name="noICPRunTillLevel" value="1" />
    
    <param name="minX" value="-1" />
    <param name="maxX" value="1" />
    <param name="minY" value="-1" />
    <param name="maxY" value="1" />
    <param name="minZ" value="0" />
    <param name="maxZ" value="2" />
    
    <param name="minFrames" value="5"/>
    
    <param name="savesClouds" value="true" />
    <param name="cloudFolder" value="/home/baxter/data/infinitam2/" />
  </node>
  
</launch>
